src/components/common/Navbar.tsx

```diff
@@ -0,0 +1,67 @@
+import { signOut } from 'next-auth/react'
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+
+
+export default function Navbar() {
+    return (
+      <div  className="sticky top-0 z-10 w-full bg-[#01003D]">
+            <div className="flex h-20 items-center justify-between px-2">
+              <div className='m-4'>
+              <Image
+                src="/fintech_logo.png"
+                alt="fintech-logo"
+                width={100}
+                height={100}
+              />
+              </div>
+                {TabConfig.map((item) => (
+                    <Tab name = {item.name} href = {item.href} >
+                    </Tab>
```

Comment:
This line of code generates warning/error for me when building. Do have a look and see whether there is any issue with this. I think you might also need to use the `Link` from `Next/Link` to route the user to the following pages.